File: Atividade_1/Carro.go
package main

import "fmt"


type Carro struct {
    Marca  string
    Modelo string
    Ano    int
}


func (c Carro) ExibirDetalhes() {
    fmt.Printf("Carro: %s %s, Ano: %d\n", c.Marca, c.Modelo, c.Ano)
}

func main() {
  
    carro1 := Carro{"Toyota", "Corolla", 2020}
    carro2 := Carro{"Honda", "Civic", 2019}
    carro3 := Carro{"Ford", "Mustang", 2021}

    
    carro1.ExibirDetalhes()
    carro2.ExibirDetalhes()
    carro3.ExibirDetalhes()
}

File: Atividade_1/Carro.java

public class Carro {
    private String marca;
    private String modelo;
    private int ano;
  
    public Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }
   
    public void exibirDetalhes() {
        System.out.println("Carro: " + marca + " " + modelo + ", Ano: " + ano);
    }

   
    public static void main(String[] args) {
        Carro carro1 = new Carro("Toyota", "Corolla", 2020);
        Carro carro2 = new Carro("Honda", "Civic", 2019);
        Carro carro3 = new Carro("Ford", "Mustang", 2021);

        carro1.exibirDetalhes();
        carro2.exibirDetalhes();
        carro3.exibirDetalhes();
    }
}
